src/formatDate.js

```diff
@@ -8,7 +8,54 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator1 = fromFormat.splice(-1, 1);
+  const separator2 = toFormat.splice(-1, 1);
```

Comment:
Using `splice` here modifies the `fromFormat` and `toFormat` arrays by removing the last element. If you need to keep the original arrays intact, consider using `slice` instead to extract the separator without modifying the arrays.